### PR TITLE
[aon_timer,dv] V1 Signoff

### PR DESCRIPTION
**Pull request by ctopal**
All action items from V1 review have been finished and PRs have been merged.

Signed-off-by: Canberk Topal <ctopal@lowrisc.org>

### CODE REVIEW

File: hw/ip/aon_timer/doc/checklist.md
Comment (sriyerg) [inline hw/ip/aon_timer/doc/checklist.md:135]: AON timer smoke tests are failing in nightly regressions:
https://reports.opentitan.org/hw/ip/aon_timer/dv/latest/results.html

Please debug / fix, thanks!

**Review by sriyerg — CHANGES_REQUESTED** [review]

File: hw/ip/aon_timer/doc/checklist.md
Comment (rswarbrick) [reply hw/ip/aon_timer/doc/checklist.md:135]: Hi @sriyerg. This should be fixed by #10331 (essentially, the commit reverts the change that extended the smoke tests to do something more exhaustive). Are you happy to remove the change request? If so, I'll wait until the nightly comes back tomorrow afternoon UK-time before merging.

Comment (rswarbrick) [reply hw/ip/aon_timer/doc/checklist.md:135]: Yep, that's the plan. I won't merge until I see a green box :-D

Comment (rswarbrick) [reply hw/ip/aon_timer/doc/checklist.md:135]: Yippee!

image

**Review by rswarbrick — APPROVED** [review]

**Pull request merged**
